### PR TITLE
[Update commons ui]Remove mutualized functions

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

**Review by flomillot — CHANGES_REQUESTED** [review]
You didn't deleted most of the functions imported in commons-ui.
fetchEnv, fetchAppsAndUrl, fetchBackEnd, etc...

**Review by flomillot — APPROVED** [review]

File: src/utils/rest-api.js [hunk 1]
```diff
@@ -10,7 +10,13 @@ import { store } from '../redux/store';
 import ReconnectingWebSocket from 'reconnecting-websocket';
 import { ContingencyListType } from './elementType';
 import { CONTINGENCY_ENDPOINTS } from './constants-endpoints';
-import { ElementType } from '@gridsuite/commons-ui';
+import {
+  ElementType,
+  fetchEnv,
+  backendFetch,
+  backendFetchJson,
+  backendFetchText, getRequestParamFromList
```
Comment (Tristan-WorkGH) [inline src/utils/rest-api.js]: prettier let this pass?
```suggestion
  backendFetchText,
  getRequestParamFromList,
```

**Review by Tristan-WorkGH — CHANGES_REQUESTED** [review]
`commons-ui` version not updated in `package.json`

**Comment by flomillot** [comment]
I forgot : need to use getToken from commons-ui too (if TL approve)

**Comment by Meklo** [comment]
> I forgot : need to use getToken from commons-ui too (if TL approve)

Done

**Review by Tristan-WorkGH — APPROVED** [review]

**Pull request merged**